### PR TITLE
Added mkdocs.

### PR DESCRIPTION
**Pull request by peci1**
A library for generating end-user documentation and its skins.

https://pypi.python.org/pypi/mkdocs
https://pypi.python.org/pypi/mkdocs-bootstrap
https://pypi.python.org/pypi/mkdocs-bootswatch
https://packages.ubuntu.com/search?keywords=mkdocs&searchon=names&suite=all&section=all
https://admin.fedoraproject.org/pkgdb/package/rpms/mkdocs
https://admin.fedoraproject.org/pkgdb/package/rpms/mkdocs-bootstrap
https://admin.fedoraproject.org/pkgdb/package/rpms/mkdocs-bootswatch/
https://packages.debian.org/search?searchon=names&keywords=mkdocs
https://packages.gentoo.org/packages/dev-python/mkdocs
https://packages.gentoo.org/packages/dev-python/mkdocs-bootstrap
https://packages.gentoo.org/packages/dev-python/mkdocs-bootswatch

### CODE REVIEW

**Comment by peci1** [comment]
@dreuter

**Comment by tfoote** [comment]
Trying to retrigger travis

**Review by tfoote — APPROVED** [review]
lgtm pending CI

**Comment by mikaelarguedas** [comment]
please hold merging, I'd like to add yakkety (and maybe wily) to it to allow source builds to keep working for people using these keys as Lunar and Kinetic will still be alive for a while

**Review by mikaelarguedas — APPROVED** [review]
lgtm thanks @peci1 ,
Great PR thanks for providing all the links and filling up all the OSes!

File: rosdep/base.yaml [hunk 1]
```diff
@@ -3439,6 +3439,91 @@ meshlab:
...
+    bionic: [mkdocs]
+    trusty:
+      pip:
+        packages: [mkdocs]
+    utopic:
+      pip:
+        packages: [mkdocs]
+    wily: [mkdocs]
+    xenial: [mkdocs]
+    yakkty: [mkdocs]
```
Comment (peci1) [inline rosdep/base.yaml]: oops

**Pull request merged**